UnicornEngine/system/source/Manager.cpp

```diff
@@ -294,5 +304,19 @@ void Manager::PollGamepads()
     }
 }
 
+void Manager::PollWindows()
+{
+    for (auto const& cit : m_windows)
+    {
+        if (cit.second->IsFocused())
```

Comment:
Not sure if maybe all windows shall be polled for events. Should not cause much overhead I dont think